doc/src/fpga_tools/fpgad/fpgad.md

```diff
@@ -5,15 +5,16 @@
 `fpgad [--socket=<sock>] [--null-bitstream=<file>]`
 
 ## DESCRIPTION ##
-```fpgad``` periodically monitors and reports the error status reflected in the device driver's error status sysfs files.
-```fpgad``` establishes the channel to communicate events to the Open Programmable Accelerator Engine (OPAE) application. 
-```fpgad``` programs a NULL bitstream in response to an AP6 (power) event. ```fpgad``` is only available on the Integrated FPGA
-Platform. You cannot run ```fpgad``` on the PCIe&reg; Accelerator Card (PAC).
+fpgad monitors the device sensors, checking for sensor values that are out of the prescribed range. 
 
-If your system does not support interrupts, you must run ```fpgad``` before the API calls `fpgaRegisterEvent` and
-`fpgaUnregisterEvent` can succeed.
+When any of the sensors is detected to be out of bounds, fpgad responds by removing the OPAE kernel driver for a configurable cool-down period.
 
-Use SIGINT to stop ```fpgad```.
+When it is being removed, the OPAE kernel driver’s shutdown path will signal any application processes that are actively using an AFU to exit, and it will reset the hardware devices, allowing them to reach a quiescent state.
```

Comment:
This statement describes the behavior of a patch that was added to the production driver some time ago.  This statement is not true of the dfl driver.

Reply:
yes, i will remove it